cellarium/cas/client.py

```diff
@@ -450,6 +479,9 @@ def __validate_cells_under_quota(self, cell_count: int) -> None:
         """
         user_quota = self.cas_api_service.get_user_quota()
         if user_quota["remaining_lifetime_quota"] is not None and cell_count > user_quota["remaining_lifetime_quota"]:
+            # If the user hasn't had a quota increase yet, show the feedback link
+            if not user_quota["quota_increased"]:
```

Comment:
As we update the db schema for user data, what happens to previous users who didn't have `quota_increased` field originally? Does the field get initialized to a default value?

Reply:
(just saw the backend migration code and Fedor's discussion, ignore this!)